java/core/src/test/java/sleeper/core/record/process/AverageRecordRateTest.java

```diff
@@ -110,6 +110,22 @@ public void shouldCalculateWithStartAndEndTimeOutsideOfAnyRuns() {
         ).containsExactly(1, 100L, 100L, Duration.ofSeconds(20), 5.0, 5.0, 10.0, 10.0);
     }
 
+    @Test
+    void shouldExcludeRunsWithRateOfNaNFromAverageRateCalculation() {
+        AverageRecordRate rate = rateFrom(
+                new RecordsProcessedSummary(
+                        new RecordsProcessed(0L, 0L),
+                        Instant.parse("2022-10-13T10:18:00.000Z"), Duration.ofSeconds(0)),
```

Comment:
It's not clear whether it's the duration of 0 or the records being 0 that's significant. Can we test the different combinations of these?